dockerfiles/Dockerfile.pip_ipython_shell

```diff
@@ -0,0 +1,12 @@
+# This install is NOT minimal!
+#   TODO: Minimize the docker container
+from python:3
+MAINTAINER x10an14
+
+WORKDIR /usr/
+
+RUN pip3 install coconut[ipython]
+
+ENTRYPOINT ["coconut"]
+CMD ["--ipython", "console"]
```

Comment:
The command `--ipython console` is _exactly_ the same as `--jupyter console`--they're just aliases of each other for people who aren't aware that IPython was renamed Jupyter. Thus, I don't think this dockerfile is necessary.